css-brochure-form/styles.css

```diff
@@ -0,0 +1,83 @@
+h3 {
+  color: #ed5f58;
+  text-align: center;
+  margin-bottom: 4px;
+  letter-spacing: 1.3px;
+}
+
+h3,
+label,
+button {
+  text-transform: uppercase;
+  font-family: 'Brandon Grotesque', serif;
+}
+
+label {
+  font-weight: 500;
+  font-size: 12px;
+  color: #444;
+  padding-left: 8px;
+}
+
+.center {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
+
+input,
+select {
+  width: 425px;
+  height: 35px;
+  border-radius: 5px;
+  border-color: #cdcbca;
```

Comment:
The only change I would make here is to set the entire border (color, style, width) so that you  don't have two different colors on the border.